emcc.py

```diff
@@ -2329,6 +2329,7 @@ def phase_linker_setup(options, state, newargs):
       # JS API directly.
       settings.REQUIRED_EXPORTS += [
         '_wasmfs_write_file',
+        '_wasmfs_getcwd',
```

Comment:
@tlively Thank you for helping me out with the error, almost all the test cases error are resolved. One last  error is the browser test case
 Expected to find '/report_result?0
' in '[no http server activity]
', diff:
This is happening on my local too. I am not able to find any solution to this browser error, neither I am able to find any workaround this.

Reply:
Sometimes that error message is caused by flaky CI. If you modify the command the test executes (which you can find with `runner --verbose`) to build to .html instead of .js, you should be able to serve the test to yourself with `emrun --no_browser <path to build directory>`. If you do that, does the test behave as expected in the browser locally?